src/Shouldly/Shouldly.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
+    <TargetFrameworks>net35;net451;netstandard1.3</TargetFrameworks>
+    <AssemblyName>Shouldly</AssemblyName>
+    <PackageId>Shouldly</PackageId>
+    <PackageTags>test;unit;testing;TDD;AAA;should;testunit;rspec;assert;assertion;framework</PackageTags>
+    <PackageIconUrl>https://raw.github.com/shouldly/shouldly/master/Icons/package_icon.png</PackageIconUrl>
+    <PackageProjectUrl>http://shouldly.github.com</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/shouldly/shouldly/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.3' ">1.6.0</NetStandardImplicitPackageVersion>
```

Comment:
Don't do this -- Remove this line and enable the default implicit ref or 1.6.1 for the netstandard 1.3 tfm. 1.6.0 contains many bugs, esp with Xamarin.